### ISSUE TITLE
implement the ASR to TensorFlow warp binding

### ISSUE THREAD

**Issue by zhoudoufu** [HawkAaron/E2E-ASR#10]
@HawkAaron Hi,
I am working on RNN-t training with this E2E-ASR repo, with pytorch binding.  and I found you have another repo that warp MXnet and TensorFlow. I suppose that warp provides universal usage for 3 different bindings.
While I do not find a plug-in place in the training code. Do I need to change pytorch method manually?  What is the right way to implement ASR training with TensorFlow warp binding?
Thanks in advance.

**Comment by HawkAaron** [comment]
You mean use tensorflow binding in this pytorch training code?

**Comment by zhoudoufu** [comment]
I would rewrite the training process in tensorflow, and use the tensorflow binding.
I am working on this actually.
Just curious, is it possible to do that (use tensorflow binding in this pytorch training code)? @HawkAaron

**Comment by HawkAaron** [comment]
Of course you can. Just convert pytorch tensor into tensorflow tensor, and call tensorflow rnnt loss.

**Comment by manish-kumar-garg** [comment]
@zhoudoufu Have a look at this [issue](https://github.com/HawkAaron/E2E-ASR/issues/14). I think you can help.
